apis/containerattached/v1beta1/containerattachedcluster_types.go

```diff
@@ -26,8 +26,7 @@ var ContainerAttachedClusterGVK = GroupVersion.WithKind("ContainerAttachedCluste
 // +kcc:proto=google.cloud.gkemulticloud.v1.AttachedCluster
 type ContainerAttachedClusterSpec struct {
 	/* The ID of the project in which the resource belongs. If it is not provided, the provider project is used. */
```

Comment:
I think by making this a required field now we should update the docs:
```suggestion
	/* The ID of the project in which the resource belongs. */
```

Reply:
It was always a required field and the docs were misleading. I have updated them.